README.md

```diff
@@ -50,7 +52,39 @@ The ingress host names are as follows, where `<base_hostname>` is your deploymen
 * Optional:
   Installation of the `podman` and [`cosign`](https://github.com/sigstore/cosign) binaries to verify that the RHTAS service is working as expected.
 
-## Deploying
+## Installation
+
+
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
+
+```
+ansible-galaxy collection install redhat.artifact_signer
+```
+
+You can also include it in a requirements.yml file and install it with ansible-galaxy collection install -r requirements.yml, using the format:
+
+
+```yaml
+collections:
+  - name: redhat.artifact_signer
+```
+
+Note that if you install any collections from Ansible Galaxy, they will not be upgraded automatically when you upgrade the Ansible package.
+To upgrade the collection to the latest available version, run the following command:
+
+```
+ansible-galaxy collection install redhat.artifact_signer --upgrade
+```
+
+You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax to install version 1.1.0:
+
+```
+ansible-galaxy collection install redhat.artifact_signer:==1.1.0
+```
+
+See [using Ansible collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
+
+
 
 1. Create an `inventory` file with a single node under the `rhtas` group:
```

Comment:
I think from this line on, we want to use the `Use Cases` section, Also, please move the `Contributing` section below `Testing`.